addon/content/components/compose/composeWidget.jsx

```diff
@@ -0,0 +1,118 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+"use strict";
+
+import { composeActions } from "../../reducer/reducer-compose.js";
+import React from "react";
+import * as ReactRedux from "react-redux";
+import PropTypes from "prop-types";
+import { TextArea, TextBox } from "./composeFields.jsx";
```

Comment:
Do we have a convention of camelCase vs hyphenated file names?

Reply:
I think I generally prefer camelCase. Though I've just realised we do have a mixture.